osdi20ae/fault_injection/README.md

```diff
@@ -0,0 +1,39 @@
+# Fault Injection artifacts
+
+This folder contains necessary components to run fault injection experiments on Theseus.
+
+## Description
+This folder contain two sub folders, which are two workloads we used during theseus fault injection experiments.
+
+1. `fs` folder contains a pre-built Theseus iso image which runs a file system workload (file open, write and read upon) booting. The image is compiled using 
+
+   `make iso THESEUS_CONFIG=" loadable unwind_exceptions use_crate_replacement use_iterative_replacement loscd_eval"`
+
+2. `ipc` folder contains a pre-built Theseus iso image which runs a two tasks communicating with each other using ipc channel (file open, write and read upon) booting. The image is compiled using 
```

Comment:
ipc channel? "ITC". 

Also you still have parentheses here about file stuff, i assume that's a copy/paste error.